cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -95,17 +98,26 @@ type indexArgs struct {
 
 	// ShardMerging is true if we want zoekt-git-index to respect compound shards.
 	ShardMerging bool
+
+	// IdBasedNames is true if we want to use ID-based names as prefix for the shard name.
+	IdBasedNames bool
```

Comment:
```suggestion
	// IDBasedNames is true if we want to use ID-based names as prefix for the shard name.
	IDBasedNames bool
```

Reply:
This is obsolete now. I removed this code.